mongodb/common.go

```diff
@@ -0,0 +1,41 @@
+package mongodb
+
+import (
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type FFSInfo struct {
+	ID    string
+	Token string
+}
+
+const (
+	key      = "ffs_info"
```

Comment:
This would require a migration if we wanted to move away from `FFS` terminology

Reply:
good call, forgot about that. will change